src/analyzer.py

```diff
@@ -138,3 +140,51 @@ def noun_phrase(input_text):
     for chunk in doc.noun_chunks:
         n_phrase_lst.append(str(chunk))
     return n_phrase_lst
+
+
+def senti_pos(tokens_column):
+    """Create column for positive words"""
+    # Start off with an empty list
+    display_series = []
+    # For each item in the already existing tokens column in the data frame,
+    for token_element in tokens_column:
+        words = create_word_list(token_element)
+        # Add the words at the end of the list to the display series since
+        # they have the most positive sentiment value
+        display_series.append(", ".join(words[len(words) - 3: len(words)]))
+    # Return an entire series based on the display_series list
+    return pd.Series(display_series)
+
+
+def senti_neg(tokens_column):
+    """Create column for negative words"""
+    # Same as senti_pos_iter
+    display_series = []
+    for token_element in tokens_column:
+        words = create_word_list(token_element)
+        # Add the words at the beginning of the list to the display series
+        # since they have the most negative sentiment value
+        display_series.append(", ".join(words[0:3]))
+
+    return pd.Series(display_series)
```

Comment:
Maybe it would be easier just to have an argument tracking whether it is `pos` or `neg`. 

If `pos`, then `display_series.append(", ".join(words[len(words) - 3: len(words)]))`

If `neg`, then `display_series.append(", ".join(words[0:3]))`